sw/device/lib/crypto/impl/hash.c

```diff
@@ -9,125 +9,57 @@
 #include "sw/device/lib/base/hardened_memory.h"
 #include "sw/device/lib/crypto/drivers/hmac.h"
 #include "sw/device/lib/crypto/drivers/kmac.h"
-#include "sw/device/lib/crypto/impl/sha2/sha256.h"
-#include "sw/device/lib/crypto/impl/sha2/sha512.h"
```

Comment:
I think these files can now be deleted.

Reply:
Do we want to keep them around, in case they are used for hardening SHA-2/HMAC through OTBN?

Reply:
Added to follow-up tracker: https://github.com/lowRISC/opentitan/issues/23191